### DISCUSSION TITLE
Jellyfin appearance for TV.

### DISCUSSION THREAD

**Discussion by MrMoreira** [jellyfin/jellyfin#6213] [General] [answered]
Hey guys, how are you! I didn't really like the appearance of the jellyfin app for android tv, I know it's hard but I would like to make a request if someone can improve it, I switched from emby to jellyfin and saw a very big difference in appearance from jellyfin in relation to emby.

I forgot to mention, that when searching for movies in the app for tv, the app leaves a little to be desired with the photos of the movies or series very small, it's a little confusing to find content, in the emby the photos are bigger and easier because of the appearance.

I forgot to mention that this is not a criticism, I love the jellyfin, but this is my opinion, and an idea to improve even more.

sorry for the english I'm using google translator

**Comment by crobibero** [comment, marked as answer]
There is already a discussion for the AndroidTV design, over in the AndroidTV repo.
https://github.com/jellyfin/jellyfin-androidtv/discussions/807
